modules/Hoot/ui/conflicts/info.js

```diff
@@ -26,19 +27,35 @@ export default class ConflictMetadata {
             /source:ingest:datetime/,
             /uuid/
         ];
+
+        Hoot.context.history().on('change.reviewtagtable',
+            _debounce( () => this.buildTagTable.bind(this)(), 300)
+        );
     }
 
     /**
      * Create tag table for revieawble items
      */
     buildTagTable() {
-        let colData    = this.data.currentFeatures,
-            tags1      = this.filterTags( colData[ 0 ] ? colData[ 0 ].tags : {} ),
+        let currentRelation;
+        try {
+            currentRelation = this.instance.graphSync.getCurrentRelation();
+        } catch (e) {
+            //current relation not found which is ok
+        }
+        //if triggered by history change and not in review mode return
+        if (!currentRelation) return;
```

Comment:
@jackgrossman18 because this method listens to history changes, it will be called outside the conflation review context and so getting the current relation may throw an "entity not found" error.